src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1407,6 +1408,9 @@ public override IOperation VisitConstructorBodyOperation(IConstructorBodyOperati
 
             EnterRegion(new RegionBuilder(ControlFlowRegionKind.LocalLifetime, locals: operation.Locals));
 
+            var member = operation.SemanticModel.GetDeclaredSymbol(operation.Syntax);
+            VisitNullChecks(operation, ((IMethodSymbol)member).Parameters, captureIdForResult);
```

Comment:
I was going to tell you to use the property on the IConstructorBodyOperation for this, then realized it doesn't exist. Filed [https://github.com/dotnet/roslyn/issues/37533](https://github.com/dotnet/roslyn/issues/37533) to follow up on that.